.env.example

```diff
@@ -8,6 +8,10 @@ ETHEREUM_SUPPORT=true
 POLKADOT_SUPPORT=true
 POLKADOT_URI=wss://polkadot.node.example/
 UI_URL=http://localhost:3000
+ARBITRUM_URI = 'https://arb1.arbitrum.io/rpc',
+ARBITRUM_RINKEBY_URI = 'https://rinkeby.arbitrum.io/rpc',
+ETHEREUM_URI = 'https://mainnet.infura.io/v3/'
+INFURA_KEY = 'adjaksdjkasjdkajsdkas'
```

Comment:
Let's just store the key in the URI, no need for another env var. I suspect you did it like that because you were testing `getDefaultProvider`?

Reply:
yes and also i didnt want to reveal the key. So if i put the whole url the key will be reveal is that okey?